definitions/digital_analytics_domain/base/ga4_events.sqlx

```diff
@@ -160,4 +160,11 @@ SELECT
 FROM ${ref('events' + dataform.projectConfig.vars.ga4_export_table_suffix)}
   -- Exclude extra day exports 
   WHERE NOT STARTS_WITH(_TABLE_SUFFIX, "intraday")
-)
\ No newline at end of file
+)
+-- Process only as much data as needed to reduce costs
+WHERE
+  TIMESTAMP_MICROS(event_timestamp) > 
+    TIMESTAMP_SUB(
+      CURRENT_TIMESTAMP(),
```

Comment:
You should put an interval considering the latest date/timestamp that you have data on the exported table. Pls, avoid using the CURRENT_TIMESTAMP().